notebooks/rskandrani.ipynb

```diff
@@ -301,8 +456,20 @@
     "def classify_sentinel_data(\n",
     "    rasters: xr.DataArray, classified_rgb_rows: ClassifiedSamples\n",
     ") -> ClassificationResult:\n",
-    "    ...  # TODO\n",
-    "    raise NotImplementedError\n"
+    "    X_train = classified_rgb_rows[\"reflectance\"].values\n",
+    "    y_train = classified_rgb_rows[\"class\"].values\n",
+    "\n",
+    "    model_rf = RandomForestClassifier().fit(X_train.T, y_train)\n",
+    "\n",
+    "    predict_classes = model_rf.predict(rasters.stack(z=(\"y\", \"x\")).values.T)\n",
```

Comment:
Naming note: "predict" looks like a verb (function) while it is a variable (name)
So you can use the preterit like `predicted_classes` instead